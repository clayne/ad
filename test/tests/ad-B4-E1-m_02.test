ad | -c always -B4 -E1 -m | endian.bin | | 0
