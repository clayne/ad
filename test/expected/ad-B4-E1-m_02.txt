[32m[K0000000000000200[m[K[36m[K:[m[K [41;1m[K0000[m[K [41;1m[K0001[m[K [41;1m[K[m[KFFFF FFFF FFFF FFFF FF[41;1m[K00[m[K [41;1m[K0000[m[K  [41;1m[K....[m[K.........[41;1m[K...[m[K
[32m[K0000000000000210[m[K[36m[K:[m[K [41;1m[K01[m[KFF FFFF FFFF FFFF FFFF FFFF FFFF [41;1m[K0000[m[K  [41;1m[K.[m[K.............[41;1m[K..[m[K
[32m[K0000000000000220[m[K[36m[K:[m[K [41;1m[K0001[m[K [41;1m[K[m[KFFFF FFFF FFFF FFFF FFFF FFFF FF[41;1m[K00[m[K  [41;1m[K..[m[K.............[41;1m[K.[m[K
[32m[K0000000000000230[m[K[36m[K:[m[K [41;1m[K0000[m[K [41;1m[K01[m[KFF FFFF FFFF FFFF FFFF FFFF FFFF  [41;1m[K...[m[K.............
[32m[K0000000000000280[m[K[36m[K:[m[K 0000 0000 [41;1m[K0000[m[K [41;1m[K0001[m[K [41;1m[K[m[K0000 FF00 0000 00[41;1m[K00[m[K  ....[41;1m[K....[m[K.......[41;1m[K.[m[K
[32m[K0000000000000290[m[K[36m[K:[m[K [41;1m[K0000[m[K [41;1m[K01[m[K00 00FF FFFF FFFF FFFF FFFF 0000  [41;1m[K...[m[K.............
[32m[K00000000000002A0[m[K[36m[K:[m[K 0000 [41;1m[K0000[m[K [41;1m[K0001[m[K [41;1m[K[m[K0000 FFFF FFFF FFFF FFFF  ..[41;1m[K....[m[K..........
[32m[K00000000000002B0[m[K[36m[K:[m[K FF00 0000 00[41;1m[K00[m[K [41;1m[K0000[m[K [41;1m[K01[m[K00 00FF FFFF FFFF  .....[41;1m[K....[m[K.......
[32m[K00000000000002C0[m[K[36m[K:[m[K FFFF FFFF 0000 0000 [41;1m[K0000[m[K [41;1m[K0001[m[K [41;1m[K[m[K0000 FFFF  ........[41;1m[K....[m[K....
[32m[K00000000000002D0[m[K[36m[K:[m[K FFFF FFFF FFFF FF00 0000 00[41;1m[K00[m[K [41;1m[K0000[m[K [41;1m[K01[m[K00  ...........[41;1m[K....[m[K.
[32m[K00000000000002E0[m[K[36m[K:[m[K 00FF FFFF FFFF FFFF FFFF 0000 0000 [41;1m[K0000[m[K  ..............[41;1m[K..[m[K
[32m[K00000000000002F0[m[K[36m[K:[m[K [41;1m[K0001[m[K [41;1m[K[m[K0000 FFFF FFFF FFFF FFFF FF00 0000  [41;1m[K..[m[K..............
[32m[K0000000000000300[m[K[36m[K:[m[K 00[41;1m[K00[m[K [41;1m[K0000[m[K [41;1m[K01[m[K00 00FF FFFF FFFF FFFF FFFF  .[41;1m[K....[m[K...........
